package.json

```diff
@@ -87,6 +87,7 @@
     "firefox-client": "0.3.0",
     "flow-bin": "0.33.0",
     "git-hooks": "1.1.1",
+    "git-hooks-win": "1.1.11",
```

Comment:
Good find! This looks really promising and it looks like it will also work on Mac. However, since the installation of this model installs a hook script, I think we also need to remove the old `git-hooks` dependency, right?